CHANGES

```diff
@@ -12,6 +12,9 @@ Incompatible changes
 
 * #4826: py domain: The structure of python objects is changed.  A boolean value
   is added to indicate that the python object is canonical one
+* #7425: MathJax: The MathJax was changed from 2 to 3. Users using a custom
+  MathJax configuration may have to set the old MathJax path or update their
+  configuration for version 3.
```

Comment:
I guess users who depends on mathjax2 do not know what they should do. It would be better to add a setting to keep using mathjax2 on math.rst and refer it on CHANGES.